src/libraries/Microsoft.PowerFx.Connectors/ConnectorHelperFunctions.cs

```diff
@@ -147,5 +151,111 @@ internal static string LogException(Exception ex)
         {
             return $"Exception {ex.GetType().FullName}, Message {ex.Message}, Callstack {ex.StackTrace}";
         }
+
+        // DType.ToString() equivalent which is safe for logging (column names are not displayed)
+        internal static string SafeDTypeToString(this DType dType)
```

Comment:
>SafeDTypeToString [](http://example.com/codeflow?start=31&length=17)

nit- "safe" is ambiguous.  Something with Logging or Anonymous would be helpful here. 
"ToAnonymousString" or "ToLoggingString.


[https://github.com/microsoft/Power-Fx/blob/6bcbe7da6a468f0f9a55585fef7fc7ecf80fdb60/src/libraries/Microsoft.PowerFx.Core/Public/CheckResult.cs#L663](https://github.com/microsoft/Power-Fx/blob/6bcbe7da6a468f0f9a55585fef7fc7ecf80fdb60/src/libraries/Microsoft.PowerFx.Core/Public/CheckResult.cs#L663)